x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -102,6 +102,7 @@ const getSavedObjectTypes = (
         is_default_fleet_server: { type: 'boolean' },
         status: { type: 'keyword' },
         unenroll_timeout: { type: 'integer' },
+        inactivity_timeout: { type: 'integer' },
```

Comment:
Q: Are we planning to query this field? If not, I'd add `index: false` to avoid mapping field explosion in the `.kibana` index.

```suggestion
        inactivity_timeout: { type: 'integer', index: false },
```

Reply:
Awesome! Thanks for confirming! Changes LGTM then